### PR TITLE
added python solution for the problem cf-1808B

### PR DESCRIPTION
**Pull request by praneethNellutla**
_Place an "x" in the corresponding checkbox if it is done or does not apply to this pull request._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions).
  - I understand that if it is clear that I have not attempted to follow these conventions, my PR will be closed.
  - If changes are requested, I will re-request a review after addressing them.
- [ ] I have linked this PR to any issues that it closes.

### CODE REVIEW

**Comment by SansPapyrus683** [comment]
why'd u close it lol

**Comment by praneethNellutla** [comment]
I didnt know what to do this is my first pull request 😅😅

**Comment by praneethNellutla** [comment]
All checks are passed but still why it isn't merged. I dont know about the process can you tell me

**Comment by SansPapyrus683** [comment]
be patient fam
the content has to be reviewed by us first

File: solutions/bronze/cf-1808B.mdx [hunk 1]
```diff
@@ -54,4 +54,40 @@ int main() {
...
+		for i in reversed(temp):
+			suf.append(prev)
+			prev += i
+
+		ans = 0
+		for i in range(n):
+			ans += temp[i] * (n - i - 1) - suf[n - i - 1]
+		res += ans
+
+	print(res)
```
Comment (SansPapyrus683) [inline solutions/bronze/cf-1808B.mdx:89]: you have a variable called `ans`, so why are you printing `res`?

**Comment by praneethNellutla** [comment]
Sorry 😅. I didn't know about all this process,

File: solutions/bronze/cf-1808B.mdx (hunk 1 above)
Comment (praneethNellutla) [reply solutions/bronze/cf-1808B.mdx:89]: Ans is for local sum and res is for global sum

Comment (SansPapyrus683) [reply solutions/bronze/cf-1808B.mdx:89]: man that's kinda misleading
switch the two variables, and name `res` something more descriptive

Comment (praneethNellutla) [reply solutions/bronze/cf-1808B.mdx:89]: Ok i will update variables with descriptive names. Do i need to close this request or should i just use git pull and then push again

Comment (SansPapyrus683) [reply solutions/bronze/cf-1808B.mdx:89]: the second one

**Review by SansPapyrus683 — APPROVED** [review]
passable

**Review by ryanchou-dev — APPROVED** [review]

**Pull request merged**